src/index.js

```diff
@@ -65,6 +65,7 @@ function createMacro(macro, options = {}) {
 function nodeResolvePath(source, basedir) {
   return resolve.sync(source, {
     basedir,
+    extensions: ['.js', '.ts', '.tsx', '.mjs'],
```

Comment:
May as well get them all 👍

```suggestion
    extensions: ['.js', '.ts', '.tsx', '.mjs', '.cjs', '.jsx'],
```